esmf_regrid/_esmf_sdo.py

```diff
@@ -61,15 +61,15 @@ def _array_to_matrix(self, array):
         We then take the transpose so that matrix multiplication happens over
         the appropriate axes.
         """
-        return array.reshape(-1, self.size, order="F").T
+        return array.T.reshape(self.size, -1)
```

Comment:
This is functionally equivalent to the call on L72, but would be nice to be consistent in how we call the numpy api. I personally prefer the explicit tuple of shape integers on L72, as it is compatible with `np.reshape`.

See notes on https://numpy.org/doc/stable/reference/generated/numpy.ndarray.reshape.html#numpy.ndarray.reshape